node_modules/col-core/lib/db.js

```diff
@@ -242,6 +263,10 @@ var setUpModel = function(sequelize) {
       'type': Sequelize.STRING,
       'allowNull': true
     },
+    'canvas_email': {
```

Comment:
Should the poller be storing the email address or do we want to wait for an explicit LTI launch for every user we send an email to ?